Update minimum iOS deployment target to iOS 9.0

tinder-owenthomas opened the pull request:
Xcode 12 no longer supports iOS 8 and throws warnings if any frameworks use a lower minimum deployment target. This PR updates the deployment target in both the podspec as well as the run tests. I didn't see any other places it needs to be updated but am happy to update other locations if they exist. 

I've included a minor version bump in the podspec since this is a semi-breaking change as it will no longer support anyone who is developing for iOS 8 - although since Apple doesn't support that type of development anyway it seems unlikely and therefore not worthy of a major bump.

tinder-owenthomas committed:
Update run tests to minimum iOS deployment target 9.0

tinder-owenthomas committed:
Update podspec to target iOS 9+ and minor version bump

tinder-owenthomas commented:
@swisspol This repo won't allow me to add you as a requested reviewer. Is there a process to get PRs like this reviewed? Just want to make sure I've done everything necessary on my end.

swisspol commented:
I’m pretty this min version is defined elsewhere as well like in the Xcode configs so they would need to be modified as well

regarding changing the version number, it should be its own PR, thanks

tinder-owenthomas committed:
Revert pod version change

tinder-owenthomas committed:
Update target to 9.0 in project.pbxproj

tinder-owenthomas commented:
> I’m pretty this min version is defined elsewhere as well like in the Xcode configs so they would need to be modified as well
> 
> regarding changing the version number, it should be its own PR, thanks

@swisspol Addressed the issues. `.pbxproj` has been updated to reflect the minimum target of 9.0 and the podspec version bump has been reverted.

tinder-owenthomas commented:
Any chance we can get these changes in for the next release @swisspol? As it is this Pod isn't compatible with Xcode 12.
